README.md

```diff
@@ -1,2 +1,6 @@
 # builder
 The image run by build pods to execute image building+pushing
+
+### Looking for contribution guidelines?
+Glad to have you here!<br>
```

Comment:
Replace the `<br>` with two spaces instead.

Reply:
Thank you! Pushed the updated file.